bears/js/JSONFormatBear.py

```diff
@@ -37,10 +38,16 @@ def run(self, filename, file,
             json_content = json.loads(''.join(file),
                                       object_pairs_hook=OrderedDict)
         except JSONDecodeError as err:
+            err_content = str(err).split(':')
```

Comment:
Problem is that those tests don't check for the actual result message. If possible write some tests that check whether your regex is working correctly^^